api/swagger-spec/v1beta3.json

```diff
@@ -9595,6 +9595,11 @@
       "type": "string",
       "description": "RFC 3339 date and time at which the object will be deleted; populated by the system when a graceful deletion is requested, read-only; if not set, graceful deletion of the object has not been requested"
      },
+     "deletionGracePeriodSeconds": {
```

Comment:
nevermind, must clean build and then regen.  swagger script doesn't also build...